src/jquery.data-remote.js

```diff
@@ -177,7 +196,7 @@
         var method = _options.oneAndDone ? 'one' : 'on';
 
         // bind to specific event type
-        $element[method](_options.eventType, function(evt) {
+        $element[method](_options.eventType, debounce(function(evt) {
```

Comment:
for instance, this would add a half second delay on a click event, which we definitely don't want, or need. I think it would be best only for events in which the normal use case calls for them to happen rapid fire, like a keyup or scroll.